### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dimsamaras**
**Please refer to the contribution guidelines in the README when submitting PRs.**

### CODE REVIEW

**Comment by whummer** [comment]
Thanks for this change @dimsamaras . This PR seems to be a duplicate of #1099 - can we close the other PR? thanks

**Comment by dimsamaras** [comment]
It is a duplicate, sorry for that! Just closed it!

File: README.md [hunk 1]
```diff
@@ -121,6 +121,30 @@ docker-compose up
 (Note that on MacOS you may have to run `TMPDIR=/private$TMPDIR docker-compose up` if
 `$TMPDIR` contains a symbolic link that cannot be mounted by Docker.)
 
+Use on existing cocker-compose project. Add in existing services. The project can be found in docker hub, no need to download or clone source:
```
Comment (dsifford) [inline README.md]: @whummer Whoops, careful on the spelling of "docker-compose". :joy:

Comment (whummer) [reply README.md]: outch! good catch, thanks

**Pull request merged**